pkg/cluster/controller.go

```diff
@@ -33,7 +36,8 @@ func runController(ctx context.Context, wg *sync.WaitGroup, dataDir string) erro
 	}
 
 	args := []string{
-		"--bind-address=0.0.0.0:11252",
+		"--bind-address=0.0.0.0",
```

Comment:
This was failing as the port number is not parseable. The correct flags need to be determined.